ethereum/jsonrpc/src/test/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcHttpServiceTest.java

```diff
@@ -2032,6 +2033,22 @@ public void ethGetStorageAtInvalidParameterStorageIndex() throws Exception {
     }
   }
 
+  @Test
+  public void assertThatLivenessProbeWorks() throws Exception {
+    try (final Response resp =
+        client.newCall(buildGetRequest(HealthService.LIVENESS_PATH)).execute()) {
```

Comment:
you main not use the constants defined in the code as values used in the tests. This would prevent you from detecting any issue on the path change for instance. Hardcoding the `/liveness` in the test seems better to me so we can check that we are not going to ship a breaking change for the user, and path change, for whatever reason, is a breaking change.